tools/markitdown/markitdown.xml

```diff
@@ -1,13 +1,13 @@
-<tool id="markitdown" name="Markitdown" version="@TOOL_VERSION@" profile="22.05">
+<tool id="markitdown" name="Markitdown" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
     <description>Convert documents to Markdown</description>
-
-    <!-- Macros inline -->
     <macros>
-        <token name="@TOOL_VERSION@">0.1.1</token>
```

Comment:
I think you can leave this here, a singel macro file for you this version is maybe too much?